packages/server/graphql/mutations/helpers/notifySlack.ts

```diff
@@ -131,25 +202,25 @@ const upsertSlackMessage = async (
     const {channel} = convoInfo
     const {latest} = channel
     if (latest) {
-      const {ts, bot_profile} = latest
-      const {name} = bot_profile
-      if (name === 'Parabol') {
-        const timestamp = new Date(Number.parseFloat(ts) * 1000)
-        const ageThresh = new Date(Date.now() - ms('5m'))
-        if (timestamp >= ageThresh) {
-          // trigger update
-          const res = await manager.updateMessage(channelId, slackText, ts)
-          if (!res.ok) {
-            console.error(res.error)
-          }
-          return
+      const {ts} = latest
+      const timestamp = new Date(Number.parseFloat(ts) * 1000)
+      const ageThresh = new Date(Date.now() - ms('5m'))
+      if (timestamp >= ageThresh) {
+        // blocks need to be stringified when updating but not posting
```

Comment:
alternatively, you could make `updateMessage` take the same type as postMessage so they have the same call signature